pkg/api/clustergroupupgrades/v1alpha1/types.go

```diff
@@ -250,8 +250,6 @@ type ClusterGroupUpgradeStatus struct {
 	PlacementBindings []string `json:"placementBindings,omitempty"`
 	// +operator-sdk:csv:customresourcedefinitions:type=status,displayName="Placement Rules"
 	PlacementRules []string `json:"placementRules,omitempty"`
-	// +operator-sdk:csv:customresourcedefinitions:type=status,displayName="Copied Policies"
```

Comment:
We can't delete it unless upversion the API. Just mark it as deprecated.

Reply:
It's a `status` field

Reply:
I don't think status fields follow different rules for versioning, do they? They are still part of the API.